File: Java/q1113.java
// Crescente e Decrescente

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
 
        Scanner input = new Scanner(System.in);
        int x = input.nextInt();
        int y = input.nextInt();
        
        while(x != y) {
            if(x < y){
                System.out.println("Crescente");
            } else {
                System.out.println("Decrescente");
            }
            
            x = input.nextInt();
            y = input.nextInt();
        }
    }
}

File: Cpp/q1113.cpp
#include <iostream>
using namespace std;

int main(){
  int n1, n2;

  while(true){
    cin >> n1 >> n2;
    if(n1 == n2){ break;}
    
    if(n1 > n2){
        cout << "Decrescente" << endl;
    } else {
        cout << "Crescente" << endl;
    } 
  }
  return 0;
}
